libs/api-interfaces/src/lib/aggregate-views/view-models/multilingual-text/language-code.enum.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Eventually, we should make the set of langauges available configurable. For
+ * now, we statically define the same set for all instances of COSCRAD.
+ *
+ * Note that we want to use the official ISO 639-3 langauge codes standard
+ * in case we need interoperability with other systems.
+ *
+ * https://iso639-3.sil.org/code_tables/639/data
+ */
+export enum LanguageCode {
```

Comment:
So cool!

Reply:
Can we add Spanish and Zapoteco (Zapotec in English, Zapoteco in Spanish) code - zap

https://en.wikipedia.org/wiki/Zapotec_languages

Reply:
Sure. Note that as soon as the select grows to ~10 menu items, or if we get complaints about noisy options, we need to make this configurable per instance.

Reply:
There is [a story](https://www.pivotaltracker.com/story/show/184431645) for this, but I decided it was premature at this point.